client/cl_gang.lua

```diff
@@ -317,6 +317,10 @@ RegisterNetEvent('qb-gangmenu:client:SocietyWithdraw', function(money)
     TriggerServerEvent('qb-gangmenu:server:withdrawMoney', withdrawAmount)
 end)
 
+RegisterNetEvent("qb-gang:client:openStash", function(gang)
+    exports.ox_inventory:openInventory('stash', { id = gang.args.gang })
+end)
```

Comment:
Should this be a local function instead of a net event? https://manason.github.io/effective-fivem-lua/structure/#prefer-limited-scope